modules/ROOT/pages/_partials/apim-metrics.adoc

```diff
@@ -0,0 +1,29 @@
+// tag::intro[]
+
+API Manager usage reports track the maximum number of xref:api-manager::latest-overview-concept.adoc#api-instances[API instances] deployed in a given month per business group, environment, and runtime. 
+
+To calculate usage, MuleSoft tracks the number of managed API instances each business group, environment, and runtime uses per hour and identifies the maximum number of managed API instances used in a month for each business group, environment, and runtime. To calculate the total count for API Manager, MuleSoft adds the maximum number of managed API instances per month used across business groups, environments, and runtimes.
```

Comment:
Any way to streamline to reduce the number of "business groups, environments, and runtimes" here?

Reply:
This content got deleted. (PM request), so this part of the partial isn't being used.